gnupg-static/plan.sh

```diff
@@ -1,7 +1,15 @@
 source ../gnupg/plan.sh
 
 pkg_name=gnupg-static
+pkg_origin=core
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_version=1.4.20
```

Comment:
Is it desirable to repeat metadata entries when they won't change from what will be inherited from `../gnupg/plan.s` ?

Reply:
Yes,  builder requires a minimal subset of metadata in order for it to connect a plan.  We also have additional metadata requirements defined in https://github.com/habitat-sh/core-plans/blob/master/bin/ci/check-default-variables.sh